database/migrations/model/Version20220907172428.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Database\Migrations\Model;
+/**
+ * Copyright 2022 OpenStack Foundation
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+use Doctrine\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema as Schema;
+use LaravelDoctrine\Migrations\Schema\Builder;
+use LaravelDoctrine\Migrations\Schema\Table;
+/**
+ * Class Version20220907172428
+ * @package Database\Migrations\Model
+ */
+class Version20220907172428 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     * @throws \Doctrine\DBAL\Schema\SchemaException
+     */
+    public function up(Schema $schema):void
+    {
+        $builder = new Builder($schema);
+        if($schema->hasTable("Presentation_Speakers") && !$builder->hasColumn("Presentation_Speakers","Order") ) {
```

Comment:
please another column name ORM could potentially generate SQL raw queries that are illegal bc Order is a reserved word

Reply:
done